reactivexcomponent/reactivexcomponent/communication/xcSession.py

```diff
@@ -0,0 +1,32 @@
+import ssl
+import websocket as WebSocket
+from reactivexcomponent.communication.publisher import Publisher
+
+
+class XcSession:
+
+    def init(self, xc_api, server_url, callback):
+        self.xc_api = xc_api
+        self.websocket = WebSocket.WebSocketApp(server_url)
+
+        def on_open(websocket):
+            callback(0, self)
```

Comment:
create constant. like SUCCESS = 0 (look for the good way to define constants in python)